web/packages/teleport/src/config.ts

```diff
@@ -98,6 +98,35 @@ const cfg = {
     AccessRequestMonthlyRequestLimit: 0,
   },
 
+  // todo mberg what defaults should be here for features not previously in this config
```

Comment:
Good question. my initial thought would be default to `false`. I'd rather someone _not_ have access when they were supposed to than the other way around. The way you have it is fine.

The only time I can see this coming up is if a newer web UI connects to an older proxy that doesn't return entitlement info. I don't see a way around this besides having a clear error message like "your proxy version doesn't report entitlements" or whatever and use a fallback (for features that existed before this) for now (with a TODO to remove in version n+1)